scripts/generate-manifest.js

```diff
@@ -128,3 +139,49 @@ function manifestLayerV2(data, hostname) {
   }
   return layer;
 }
+
+function manifestLayerV6(data, hostname, opts) {
+  const fields = data.fieldMapping.map(fieldMap => ({
+    type: fieldMap.type,
+    id: fieldMap.name,
+    label: { ...{ en: fieldMap.desc }, ...getFieldLabels(fieldMap.name, opts.fieldInfo) },
+  }));
+  const layer = {
+    layer_id: data.name,
+    created_at: data.createdAt,
+    attribution: data.attribution,
+    formats: data.emsFormats.map(format => {
+      return {
+        format: format.type,
+        url: `https://${hostname}/files/${format.file}?elastic_tile_service_tos=agree`,
+        legacy_default: format.default || false,
+      };
+    }),
+    fields: fields,
+    layer_name: data.humanReadableName,
+  };
+  return layer;
+}
+
+function getDefaultFormat(emsFormats) {
+  return emsFormats.filter(format => format.default);
+}
+
+function getFieldLabels(fieldName, fieldInfo) {
+  if (fieldName.startsWith('label_') && _.get(fieldInfo, 'name.i18n')) {
+    const lang = fieldName.replace('label_', '');
+    const labels = _.mapValues(fieldInfo.name.i18n, (label) => {
```

Comment:
```suggestion
    const labels = _.mapValues(fieldInfo.name.i18n, label => `${label} (${lang})`);
```

Note that `${label} (${lang})` is not a good translation string -- because in some languages it would look horrible, especially in RTL.  Usually, the formatting string itself is also stored in some translation system, so that users can provide locale-specific formatting, e.g. `label [lang]` or something funkier. This probably should be a separate patch though, once the translation method is established.

Reply:
Agreed we could patch this separately. It wouldn't impact 6.6 users as this is just a placeholder for now. fwiw, I do agree this will be a longer term issue that we need to resolve (and I'm the wrong person to give meaningful feedback on this topic).

Reply:
As we start to support additional languages we should see how this is done in Kibana and patch the manifest as necessary. 👍